### PR TITLE
Aggregate backup in neo4j-admin helm chart

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1719.surge.sh

Updated pages:
- [Back up and restore (online)](https://neo4j-docs-operations-1719.surge.sh/operations-manual/5/kubernetes/operations/backup-restore)

File: modules/ROOT/pages/kubernetes/operations/backup-restore.adoc [hunk 1]
```diff
@@ -597,6 +612,142 @@ This pod performs a backup of one or multiple databases, a consistency check of
 . Monitor the backup pod logs using `kubectl logs pod/<neo4j-backup-pod-name>` to check the progress of the backup.
 . Check that the backup files and the consistency check reports have been uploaded to the cloud provider bucket or on-premises storage.
 
+[role=label--new-5.21]
+[[kubernetes-neo4j-backup-aggregate]]
+== Aggregate a database backup chain
+
+The aggregate command turns a backup chain into a single backup file.
```
Comment (harshitsinghvi22) [inline modules/ROOT/pages/kubernetes/operations/backup-restore.adoc:619]: the aggregate backup command

File: modules/ROOT/pages/kubernetes/operations/backup-restore.adoc [hunk 2]
```diff
@@ -597,6 +612,142 @@ This pod performs a backup of one or multiple databases, a consistency check of
...
+----
+======
+=====
+. Install the _neo4j-admin_ Helm chart using the configured _backup-values.yaml_ file:
++
+[source, shell, role='noheader']
+----
+helm install backup-name neo4j-admin -f /path/to/your/backup-values.yaml
+----
+. Monitor the backup pod logs using `kubectl logs pod/<neo4j-backup-pod-name>` to check the progress of the backup.
```
Comment (harshitsinghvi22) [inline modules/ROOT/pages/kubernetes/operations/backup-restore.adoc:747]: pod/<neo4j-aggregate-backup-pod-name> progress of aggregate backup

**Review by harshitsinghvi22 — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/kubernetes/operations/backup-restore.adoc (hunk 2 above)
Comment (harshitsinghvi22) [reply modules/ROOT/pages/kubernetes/operations/backup-restore.adoc:747]: neo4j-backup-pod-name --> neo4j-aggregate-backup-pod-name

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1719.surge.sh

Updated pages:
- [Back up and restore (online)](https://neo4j-docs-operations-1719.surge.sh/operations-manual/5/kubernetes/operations/backup-restore)

File: modules/ROOT/pages/kubernetes/operations/backup-restore.adoc [hunk 3]
```diff
@@ -597,6 +612,142 @@ This pod performs a backup of one or multiple databases, a consistency check of
...
+----
+======
+=====
+. Install the _neo4j-admin_ Helm chart using the configured _backup-values.yaml_ file:
++
+[source, shell, role='noheader']
+----
+helm install backup-name neo4j-admin -f /path/to/your/backup-values.yaml
+----
+. Monitor the pod logs using `kubectl logs pod/<neo4j-backup-pod-name>` to check the progress of aggregate backup.
```
Comment (harshitsinghvi22) [inline modules/ROOT/pages/kubernetes/operations/backup-restore.adoc:747]: neo4j-backup-pod-name --> neo4j-aggregate-backup-pod-name

File: modules/ROOT/pages/kubernetes/operations/backup-restore.adoc [hunk 4]
```diff
@@ -597,6 +612,142 @@ This pod performs a backup of one or multiple databases, a consistency check of
...
+      enabled: true
+      verbose: false
+      keepOldBackup: false
+      parallelRecovery: false
+      fromPath: "s3://bucket1/bucket2"
+      # Database name to aggregate. Can contain * and ? for globbing.
+      database: "neo4j"
+
+#The service account must already exist and have the necessary permissions to access and manage your S3 bucket. See https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies.html for more information.
+serviceAccountName: "my-service-account"
```
Comment (harshitsinghvi22) [inline modules/ROOT/pages/kubernetes/operations/backup-restore.adoc:697]: we should add the permissions required in the policy for the aggregate backup to work. Have shared the required permissions in the demo video. either we add those here or add them in the aggregate backup documentation and reference it here

**Review by harshitsinghvi22 — CHANGES_REQUESTED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1719.surge.sh

Updated pages:
- [Back up, aggregate, and restore (online)](https://neo4j-docs-operations-1719.surge.sh/operations-manual/5/kubernetes/operations/backup-restore)

File: modules/ROOT/pages/kubernetes/operations/backup-restore.adoc (hunk 3 above)
Comment (harshitsinghvi22) [reply modules/ROOT/pages/kubernetes/operations/backup-restore.adoc:767]: neo4j-backup-pod-name --> neo4j-aggregate-backup-pod-name

**Review by harshitsinghvi22 — CHANGES_REQUESTED** [review]

**Review by harshitsinghvi22 — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1719.surge.sh

Updated pages:
- [Back up, aggregate, and restore (online)](https://neo4j-docs-operations-1719.surge.sh/operations-manual/5/kubernetes/operations/backup-restore)

**Pull request merged**